README.md

```diff
@@ -9,3 +9,20 @@ https://www.theodinproject.com/courses/ruby-programming/lessons/oop
 
 # Project Description
 Learning OOP with Ruby
+
+#Instructions
+The object of Tic Tac Toe is to get three in a row. You play on a three by three game board. The first player will choose to play as 'X' or 'O'. Players alternate placing Xs and Os on the game board until either oppent has three in a row or all nine squares are filled. Player one goes first, and in the event that no one has three in a row, the stalemate ends up in a draw.
+In order to fill the board, players must select the coordinates to put their marks. First coordinate number refers to the line, and the second coordinate number refers to the column.
+
+     0       1       2   
+         |       |       
```

Comment:
**[OPTIONAL]** Good readme file overall, but it is lacking some styling, it could be improved, please, find some inspiration [here](https://github.com/othneildrew/Best-README-Template) or in this [course](https://www.udacity.com/course/writing-readmes--ud777)